server/events/models.py

```diff
@@ -0,0 +1,77 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.contrib.auth import models as auth_models
+from django.db import models
+
+# Create your models here.
+class Event(models.Model):
+    """ A representation of an ACC event.
```

Comment:
Nitpick: Try to keep class-level comments consistent with the `variable`:  `description` format.

Reply:
fixed